setup/account_invoice_sent_flag/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
+)
```

Comment:
These are from another module, the shouldn't be here either. There's more lines like this in the next files, please remove all of them